tests/unit/authors/test_authors_receivers.py

```diff
@@ -63,6 +77,37 @@ def test_name_variations():
             'Richard Ellis'])
 
 
+@mock.patch('requests.post', return_value=BeardAPI())
+def test_phonetic_block_generation(mock):
+    from flask import current_app
+
+    json_dict = {
+        "authors": [{
+            "full_name": "Grzegorz Jacenków"
+        }, {
+            "full_name": "**BROKEN**",
+        }, {
+            "full_name": "John Richard Ellis"
+        }]
+    }
+
+    # In order to run the method, BEARD_API_URL value must be set.
+    temp_config = current_app.config.get('BEARD_API_URL')
+    current_app.config['BEARD_API_URL'] = True
```

Comment:
`True` is technically not the right value as it should really be an URL string, but for your case it works I guess.

Btw, this is a neat trick to avoid reverting manually: https://github.com/inspirehep/inspire-next/blob/master/tests/integration/workflows/test_arxiv_workflow.py#L252-L259